### PR TITLE
Issue#20877 Updated Interactive Beam README

### LINKED ISSUE #20877: README example for Beam Python InteractiveRunner is broken
Running the example from the interactive Beam README doc [1] produces an error:

```

import apache_beam as beam
from apache_beam.runners.interactive.interactive_runner import InteractiveRunner

p
= beam.Pipeline(InteractiveRunner())
pcoll = p | beam.Create([1, 2, 3])
result = p.run().wait_until_finish()
pcoll_list
= result.get(pcoll)  # This returns a list!

```

Results in:
```

---------------------------------------------------------------------------
AttributeError       
                    Traceback (most recent call last)
<ipython-input-11-b3606ce83458> in <module>()

     2 pcoll = p | beam.Create([1, 2, 3])
      3 result = p.run().wait_until_finish()
----> 4 pcoll_list
= result.get(pcoll)  # This returns a list!

AttributeError: 'NoneType' object has no attribute 'get'

```

Side note: I had to look-up the import statements here. As long as you're updating the example, it would be nice for it to be runnable with copy & paste.

[1] https://github.com/apache/beam/blob/master/sdks/python/apache_beam/runners/interactive/README.md 

 

Imported from Jira [BEAM-12234](https://issues.apache.org/jira/browse/BEAM-12234). Original Jira may contain additional context.
Reported by: shoyer.

### PR DESCRIPTION
**Pull request by KevinGG**
fixes #20877

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [x] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: sdks/python/apache_beam/runners/interactive/README.md [hunk 1]
```diff
@@ -27,38 +27,142 @@ exploration much faster and easier. It provides nice features including
 
 1.  Graphical representation
 
-    When a pipeline is executed on a Jupyter notebook, it instantly displays the
-    pipeline as a directed acyclic graph. Sampled PCollection results will be
-    added to the graph as the pipeline execution proceeds.
+    Pipeline DAG can be visualized through:
```
Comment (rohdesamuel) [inline sdks/python/apache_beam/runners/interactive/README.md:30]: Use a more active voice for the comments. For example this line could read "Visualize the Pipeline DAG:". And "Visualize elements in a PCollection:"

**Review by rohdesamuel — APPROVED** [review]
Very excellent code snippets and explanations! Just a grammar nit, but otherwise lgtm

File: sdks/python/apache_beam/runners/interactive/README.md (hunk 1 above)
Comment (KevinGG) [reply sdks/python/apache_beam/runners/interactive/README.md]: Thanks! Made the amendments.

**Review by rohdesamuel — APPROVED** [review]
lgtm! thanks!

**Pull request merged**